challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{|x| x.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  [].push input.find{|x| x == "dog"}
```

Comment:
This works, but there's an enumerable that does this without having to push it onto an empty array.